content/sensu-go/5.5/reference/tessen.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Tessen"
+description: "Tessens sends anonymized data about the Sensu instance to Sensu Inc. You can use sensuctl to view and manage Tessen configuration. Read the reference to configure Tessen."
+weight: 10
+version: "5.5"
+product: "Sensu Go"
+menu: 
+  sensu-go-5.5:
+    parent: reference
+---
+
+- [Configuring Tessen](#configuring-tessen)
+- [Specification](#tessen-specification)
+- [Example](#tessen-example)
+
+## What is Tessen?
+
+Tessen is the Sensu call-home service.
+Included with every Sensu backend, Tessen sends anonymized data about the Sensu instance to Sensu Inc. every six hours, including:
```

Comment:
Sensu Go default interval is 30 minutes/1800 seconds: https://github.com/sensu/tessen/blob/6b14535c123d4416aee6594e1351ff89fd041030/template.yml#L163